### DISCUSSION TITLE
Backup HA su Google Drive

### DISCUSSION THREAD

**Discussion by Ravo86** [indomus/forum#2578] [HUB: Home Assistant]
Ciao, sono riuscito a creare dei backup schedulati per il mio HA Core ma al momento rimangono in locale sul mio Raspberry. Qualcuno ha già integrato uno script per caricare la cartella di backup su GDrive?

**Comment by snakuzzo** [comment]
Google > cerca "linux upload file to google drive"

Troverai tanta roba per farlo
